src/app/candidate-form/candidate-form.component.ts

```diff
@@ -0,0 +1,30 @@
+import { Component, OnInit } from '@angular/core';
+import { Candidate } from '../candidate';
+
+@Component({
+  selector: 'app-candidate-form',
+  templateUrl: './candidate-form.component.html',
+  styleUrls: ['./candidate-form.component.css']
+})
+export class CandidateFormComponent implements OnInit {
+
+  constructor() { }
+
+  ngOnInit(): void {
+    console.log(this.model);
+  }
+
+  educationLevel = ['High School', 'College/University', 'Other',];
+  model = new Candidate('name', 'label' ,'email', 'phone', 'website', 'github', 'twitter', this.educationLevel[0], 'schoolName');
+  submitted = false;
+
+  onSubmit() {
+    console.log("Building JSON:");
+    console.log(this.model); //Incomplete JSON Resume Object.
+    this.submitted = true;
```

Comment:
This is where you are console logging the JSON object it seems. I'm not very familiar with Angular unfortunately, so I'm wondering where the submit button is. Some comments would be helpful but I know the time constraint makes it difficult.